SConscript

```diff
@@ -1,40 +1,40 @@
 Import('env', 'envCython', 'arch', 'common')
 
 
-visionipc_dir = Dir('visionipc')
+visionipc_dir = Dir('msgq/visionipc')
 gen_dir = Dir('gen')
 
 
-# Build messaging
-
-
-messaging_objects = env.SharedObject([
-  'messaging/messaging.cc',
-  'messaging/event.cc',
-  'messaging/impl_zmq.cc',
-  'messaging/impl_msgq.cc',
-  'messaging/impl_fake.cc',
-  'messaging/msgq.cc',
+# Build msgq
+msgq_objects = env.SharedObject([
+  'msgq/ipc.cc',
+  'msgq/event.cc',
+  'msgq/impl_zmq.cc',
+  'msgq/impl_msgq.cc',
+  'msgq/impl_fake.cc',
+  'msgq/msgq.cc',
 ])
-messaging = env.Library('messaging', messaging_objects)
-messaging_python = envCython.Program('messaging/messaging_pyx.so', 'messaging/messaging_pyx.pyx', LIBS=envCython["LIBS"]+[messaging, "zmq", common])
-
+msgq = env.Library('msgq', msgq_objects)
+msgq_python = envCython.Program('msgq/ipc_pyx.so', 'msgq/ipc_pyx.pyx', LIBS=envCython["LIBS"]+[msgq, "zmq", common])
 
 # Build Vision IPC
-vipc_files = ['ipc.cc', 'visionipc_server.cc', 'visionipc_client.cc', 'visionbuf.cc']
+vipc_files = ['visionipc.cc', 'visionipc_server.cc', 'visionipc_client.cc', 'visionbuf.cc']
 vipc_sources = [f'{visionipc_dir.abspath}/{f}' for f in vipc_files]
 
 if arch == "larch64":
   vipc_sources += [f'{visionipc_dir.abspath}/visionbuf_ion.cc']
 else:
   vipc_sources += [f'{visionipc_dir.abspath}/visionbuf_cl.cc']
 
+print(f'Building Vision IPC with {vipc_sources}')
```

Comment:
fyi you don't want to print in a SCons fi because it'll spam every time
```
batman:sixpilot$ scons
scons: Reading SConscript files ...
Building Vision IPC with ['/dev/shm/openpilot/sixpilot/msgq_repo/msgq/visionipc/visionipc.cc', '/dev/shm/openpilot/sixpilot/msgq_repo/msgq/visionipc/visionipc_server.cc', '/dev/shm/openpilot/sixpilot/msgq_repo/msgq/visionipc/visionipc_client.cc', '/dev/shm/openpilot/sixpilot/msgq_repo/msgq/visionipc/visionbuf.cc', '/dev/shm/openpilot/sixpilot/msgq_repo/msgq/visionipc/visionbuf_cl.cc']
Building Vision IPC with ['msgq/visionipc/visionipc.os', 'msgq/visionipc/visionipc_server.os', 'msgq/visionipc/visionipc_client.os', 'msgq/visionipc/visionbuf.os', 'msgq/visionipc/visionbuf_cl.os']
scons: done reading SConscript files.
scons: Building targets ...
scons: `.' is up to date.
scons: done building targets.
```